lighthouse-core/config/default-config.js

```diff
@@ -384,6 +392,18 @@ const defaultConfig = {
       title: str_(UIStrings.seoCrawlingGroupTitle),
       description: str_(UIStrings.seoCrawlingGroupDescription),
     },
+    'best-practices-safety': {
+      title: str_(UIStrings.bestPracticesSafetyGroupTitle),
+    },
+    'best-practices-ux': {
+      title: str_(UIStrings.bestPracticesUXGroupTitle),
+    },
+    'best-practices-browser': {
```

Comment:
`best-practices-compat` if we're abbreviating? could also just be more verbose for this one `best-practices-browser-compat`

browser feels a bit too vague since all of this in a browser :)